test/crd_checks.go

```diff
@@ -20,185 +20,11 @@ limitations under the License.
-func WaitForRevisionState(client *ServingClients, name string, inState func(r *v1alpha1.Revision) (bool, error), desc string) error {
-	span := logging.GetEmitableSpan(context.Background(), fmt.Sprintf("WaitForRevision/%s/%s", name, desc))
-	defer span.End()
-
-	var lastState *v1alpha1.Revision
-	waitErr := wait.PollImmediate(interval, timeout, func() (bool, error) {
-		var err error
-		lastState, err = client.Revisions.Get(name, metav1.GetOptions{})
-		if err != nil {
-			return true, err
-		}
-		return inState(lastState)
-	})
-
-	if waitErr != nil {
-		return errors.Wrapf(waitErr, "revision %q is not in desired state, got: %+v", name, lastState)
-	}
-	return nil
-}
-
-// CheckRevisionState verifies the status of the Revision called name from client
-// is in a particular state by calling `inState` and expecting `true`.
-// This is the non-polling variety of WaitForRevisionState
-func CheckRevisionState(client *ServingClients, name string, inState func(r *v1alpha1.Revision) (bool, error)) error {
-	r, err := client.Revisions.Get(name, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-	if done, err := inState(r); err != nil {
-		return err
-	} else if !done {
-		return fmt.Errorf("revision %q is not in desired state, got: %+v", name, r)
-	}
-	return nil
-}
-
-// WaitForServiceState polls the status of the Service called name
-// from client every `interval` until `inState` returns `true` indicating it
-// is done, returns an error or timeout. desc will be used to name the metric
-// that is emitted to track how long it took for name to get into the state checked by inState.
-func WaitForServiceState(client *ServingClients, name string, inState func(s *v1alpha1.Service) (bool, error), desc string) error {
-	span := logging.GetEmitableSpan(context.Background(), fmt.Sprintf("WaitForServiceState/%s/%s", name, desc))
-	defer span.End()
-
-	var lastState *v1alpha1.Service
-	waitErr := wait.PollImmediate(interval, timeout, func() (bool, error) {
-		var err error
-		lastState, err = client.Services.Get(name, metav1.GetOptions{})
-		if err != nil {
-			return true, err
-		}
-		return inState(lastState)
-	})
-
-	if waitErr != nil {
-		return errors.Wrapf(waitErr, "service %q is not in desired state, got: %+v", name, lastState)
-	}
-	return nil
-}
-
-// CheckServiceState verifies the status of the Service called name from client
-// is in a particular state by calling `inState` and expecting `true`.
-// This is the non-polling variety of WaitForServiceState.
-func CheckServiceState(client *ServingClients, name string, inState func(s *v1alpha1.Service) (bool, error)) error {
-	s, err := client.Services.Get(name, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-	if done, err := inState(s); err != nil {
-		return err
-	} else if !done {
-		return fmt.Errorf("service %q is not in desired state, got: %+v", name, s)
-	}
-	return nil
-}
-
 // GetConfigMap gets the knative serving config map.
```

Comment:
Seems to be more of a k8s_checks and getters now that we split this?

Reply:
Yeah, I'd rather leave it alone in this change, since I'm not already moving it, but I agree with the cleanup idea.  In general, I was trying to bring sanity to some of the file naming as I moved things because I feel like I can never find stuff in `test/` :(